packages/test/test-end-to-end-tests/src/test/gc/gcInactiveNodes.spec.ts

```diff
@@ -34,13 +35,15 @@ describeNoCompat("GC inactive nodes tests", (getTestObjectProvider) => {
         summaryOptions: { disableSummaries: true },
         gcOptions: { gcAllowed: true, deleteTimeoutMs },
     };
+    const innerRequestHandler = async (request: IRequest, runtime: IContainerRuntimeBase) =>
+        runtime.IFluidHandleContext.resolveHandle(request);
     const runtimeFactory = new ContainerRuntimeFactoryWithDefaultDataStore(
         dataObjectFactory,
         [
             [dataObjectFactory.type, Promise.resolve(dataObjectFactory)],
         ],
         undefined,
-        undefined,
+        [innerRequestHandler],
```

Comment:
What's this and the below change about?

Reply:
With this change, the revived and changed event are logged only for summarizer clients.
Previously, these tests created a regular client and were summarizing that by getting the container runtime and calling summarization APIs. So, it had to be changed to create a summarizer client instead.

The reason for adding the `innerRequestHandler` is that without it, you can only request "/" or "" from the container runtime. The `loadSummarizer` function requests "default" which doesn't work.